docs/faq.md

```diff
@@ -71,9 +71,13 @@ readability because operators are misaligned. Disable W503 and enable the
 disabled-by-default counterpart W504. E203 should be disabled while changes are still
 [discussed](https://github.com/PyCQA/pycodestyle/issues/373).
 
-## Does Black support Python 2?
+## Which Python versions does Black support?
 
-Support for formatting Python 2 code was removed in version 22.0.
+_Black_ supports at least all Python versions that have not reached their end of life.
```

Comment:
This FAQ entry doesn't actually answer the question: we currently support 3.6 through 3.10.

"Supporting" a Python version in Black can mean two things: running Black under that version, or formatting all code that's valid for that version. Those two meanings are the same right now, but they weren't in the past (we supported formatting Python 2 but not running under Python 2; we supported running under 3.10 before we could fully format 3.10) and they might not be again in the future. Maybe we can make this more explicit too.